common/shlibs

```diff
@@ -2477,6 +2477,7 @@ libanthygobject-1.0.so.5 ibus-anthy-1.5.6_1
 libtbbmalloc_proxy.so.2 tbb-4.3_1
 libtbbmalloc.so.2 tbb-4.3_1
 libtbb.so.12 tbb-2021.11.0_1
+libembree4.so.4 embree-4.3.3_1
 libembree.so.3 embree-3.12.2_1
```

Comment:
remove the old libembree entries:
```
$ xlocate libembree
embree-4.3.3_1        /usr/lib/libembree4.so.4
embree-devel-4.3.3_1  /usr/lib/libembree4.so -> /usr/lib/libembree4.so.4
```

Reply:
Removed the old entries from the shlibs.